cicd/pipelines/customizing-configurations-in-the-tektonconfig-cr.adoc

```diff
@@ -0,0 +1,61 @@
+:_content-type: ASSEMBLY
+[id="customizing-configurations-in-the-tektonconfig-cr"]
+= Customizing configurations in the TektonConfig custom resource
+include::_attributes/common-attributes.adoc[]
+:context: customizing-configurations-in-the-tektonconfig-cr
+
+toc::[]
+
+[role="_abstract"]
+You can customize the following {pipelines-title} behaviors by using the `TektonConfig` custom resource (CR):
+
+* Configuring the {pipelines-title} control plane
+* Changing the default service account
+* Disabling the service monitor
+* Configuring the pipeline resolvers
+* Disabling cluster tasks and pipeline templates
+* Disabling the integration of Tekton Hub
+* Disabling the automatic creation of RBAC resources
+* Pruning of task run and pipeline run
+
+[discrete]
+== Prerequisites
+
+* You have installed the {pipelines-title} Operator.
+
+include::modules/op-configuring-pipelines-control-plane.adoc[leveloffset=+1]
+
+include::modules/op-configuration-options-with-default-values.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+* See xref:../../cicd/pipelines/authenticating-pipelines-using-git-secret.adoc#op-configuring-ssh-authentication-for-git_authenticating-pipelines-using-git-secret[Configuring SSH authentication for Git] for more information about Git SSH secrets.
+
+include::modules/op-optional-configuration-options.adoc[leveloffset=+2]
+
+include::modules/op-changing-default-service-account.adoc[leveloffset=+2]
+
+include::modules/op-disabling-the-service-monitor.adoc[leveloffset=+2]
+
+include::modules/op-configuring-pipeline-resolvers.adoc[leveloffset=+2]
+
+include::modules/op-disabling-cluster-tasks-and-pipeline-templates.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* To learn more about cluster tasks, see xref:managing-nonversioned-and-versioned-cluster-tasks.adoc#managing-nonversioned-and-versioned-cluster-tasks[Managing non-versioned and versioned cluster tasks]
+* For more information on how to use custom pipeline templates, see xref:../../cicd/pipelines/working-with-pipelines-using-the-developer-perspective.adoc#using-custom-pipeline-template-for-git-import_working-with-pipelines-using-the-developer-perspective[Using a custom pipeline template for creating and deploying an application from a Git repository]
+
+include::modules/op-disabling-the-integretion-of-tekton-hub.adoc[leveloffset=+1]
+
+include::modules/op-disabling-automatic-creation-of-rbac-resources.adoc[leveloffset=+1]
+
+include::modules/op-automatic-pruning-taskrun-pipelinerun.adoc[leveloffset=+1]
+
+include::modules/op-default-pruner-configuration.adoc[leveloffset=+2]
```

Comment:
IMHO it's time to remove `Starting with Red Hat OpenShift Pipelines 1.6, auto-pruning is enabled by default.` in this module.

Reply:
Done